.github/templates/main.yml

```diff
@@ -37,6 +37,15 @@ with:
   retention-days: 1
 #@ end
 
+#@ def uploadArtifacts(titleName, uploadName, relPath, retentionDays = "${{ github.event_name != 'pull_request' && 30 || 1 }}"):
```

Comment:
We can use `uploadName` instead of `titleName` - it's probably not going to be as visually appealing (e.g. `Store artifacts for dashboard.charts` rather than `Store artifacts for Charts Dashboard`), but that seems like a minor inconvenience.

Reply:
I actually went like that at the beginning but then I thought it looked a bit too long. But if you think it's fine, then I'll change it to how it was before.